discojs/src/types/data_format.ts

```diff
@@ -0,0 +1,32 @@
+import { List } from "immutable";
+
+import type { Image, processing, Tabular, Text } from "../index.js";
+
+/** what get's ingested by Disco */
```

Comment:
```suggestion
/** what gets ingested by Disco */
```
Same for the other comments